DependencyInjection/MonologExtension.php

```diff
@@ -57,6 +58,7 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setParameter('monolog.use_microseconds', $config['use_microseconds']);
 
             // always autowire the main logger, require Symfony >= 2.8
+            $container->setAlias('Psr\Log\LoggerInterface', new Alias('monolog.logger', false));
```

Comment:
I would put this directly in the XML file.

Btw, I would put the autowiring point to ``logger`` itself (making it easier for people replacing the main logger service)